internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -0,0 +1,61 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+
+	mssql "github.com/denisenkom/go-mssqldb"
+)
+
+type SingleUseConnector struct {
+	logger log.Logger
+}
+
+func (connector *SingleUseConnector) Connect(
+	clientConn net.Conn,
+	credentialValuesByID connector.CredentialValuesByID,
+) (net.Conn, error) {
+
+	// Using DSN (Data Source Name) string because gomssql forces us to.
+	//
+	// NOTE: Secretless has some unfortunate naming collisions with the
+	// go-mssqldb driver package.  The driver package has its own concept of a
+	// "connector", and its connectors also have a "Connect" method.
+	driverConnector, err := mssql.NewConnector(
+		fmt.Sprintf(
+			"sqlserver://%s:%s@%s",
+			credentialValuesByID["user"],
+			credentialValuesByID["password"],
+			credentialValuesByID["address"],
```

Comment:
Do these need escaping? If so, we either need a fix for it or a new issue opened